var/spack/repos/builtin/packages/hdf5/package.py

```diff
@@ -61,31 +64,35 @@ class Hdf5(AutotoolsPackage):
     variant('java', default=False, description='Enable Java support')
     variant('threadsafe', default=False,
             description='Enable thread-safe capabilities')
-
+    variant('tools', default=False, description='Enable build tools')
     variant('mpi', default=True, description='Enable MPI support')
     variant('szip', default=False, description='Enable szip support')
+    variant('zlib', default=True, description='Enable zlib support')
     variant('pic', default=True,
             description='Produce position-independent code (for shared libs)')
-    # Build HDF5 with API compaitibility.
-    variant('api', default='none', description='choose api compatibility', values=('none', 'v114', 'v112', 'v110', 'v18', 'v16'), multi=False)
+    # Build HDF5 with API compatibility.
+    variant('api', default='none', description='Choose api compatibility for earlier version', values=('none', 'v114', 'v112', 'v110', 'v18', 'v16'), multi=False)
```

Comment:
Instead of `none` as the default (since there is still an API even when none is selected), how about `default`, `auto`, `native`, ...?

Reply:
Would 'current' as the default convey the meaning that the default api is that of the installed version?

Reply:
Most likely we'll go with 'default'